pkg/metadata/metadata.go

```diff
@@ -136,6 +143,21 @@ func RemoveRebootAnnotation(ctx context.Context, c client.Client, node core.Node
 	return nil
 }
 
+// RemoveBlockedLabel clears the blocked label from the given Node
```

Comment:
nit: mention that this is a no-op if already not present in the func doc (or change name to `ensureBlockedLabelIsRemoved`)